index.bs

```diff
@@ -406,73 +413,154 @@ When this method is invoked, the user agent MUST execute the following algorithm
     value. If {{ScopedCredentialOptions/timeout}} was not specified, then set |adjustedTimeout| to a platform-specific
     default.
 
-2. Let |promise| be [=a new Promise=]. Return |promise| and start a timer for |adjustedTimeout| milliseconds.
-    Then asynchronously continue executing the following steps. If any fatal error is encountered in this process other than the
-    ones enumerated below, cancel the timer, reject |promise| with a DOMException whose name is "{{UnknownError}}", and terminate
-    this algorithm.
+    Issue: Put some constraints on the "reasonable range".
 
 3. Set |callerOrigin| to the <a>current settings object</a>'s <a>origin</a>. If |callerOrigin| is
```

Comment:
Though, as @jyasskin notes in https://github.com/w3c/webauthn/pull/350#discussion_r101831398
> ...Markdown is just as happy starting every numbered list item with "1.", so that new list items don't cause a diff for every subsequent item.